views/js/controller/SystemStatus/tpl/chartsContainer.tpl

```diff
@@ -0,0 +1,10 @@
+<div class="system-status__charts-container">
+    <div>
+        <div class="system-status__chart-title">{{__ "Used space on ElastiCache storage"}}</div>
```

Comment:
Details should be given from check report.
There may be more/less than two reports reports of pie type.